setup.py

```diff
@@ -2,7 +2,7 @@
 # or saved.  Do not modify them directly here.
 # NB: PACKAGES is deprecated
 NAME = "ZenPacks.zenoss.OpenStackInfrastructure"
-VERSION = "2.4.0"
+VERSION = "2.5.0dev"
```

Comment:
We don't use `dev` suffix no more, according to https://zenpackers.readthedocs.io/en/latest/zenpack_versioning.html